pkg/kubeutils/kubeutils.go

```diff
@@ -210,6 +220,44 @@ func writeStatusMessage(writer *spinner.MessageWriter, install embeddedclusterv1
 	}
 }
 
+func WaitForHAMigration(ctx context.Context, cli client.Client) error {
+	// TODO: determine an appropriate timeout
+	backoff := wait.Backoff{Steps: 60, Duration: 5 * time.Second, Factor: 1.0, Jitter: 0.1}
```

Comment:
Open to suggestions for how long we should wait for the HA migration.

Reply:
an exponential backoff here might not be a good idea for migrations that will take a very long time. i would suggest:

- for loop without a timeout (similar to how we wait for velero restores to complete)
- fixed sleep period (5s?)

Reply:
@sgalsaleh thanks for the feedback - definitely makes sense to avoid extra waiting

Reply:
Addressed in https://github.com/replicatedhq/embedded-cluster/pull/657/commits/53071a1b1830fc69b0f366120e284ff8a273fe93